ffxivbot/pika_rabbit.py

```diff
@@ -485,6 +485,14 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
                         # return
                     group_commands = json.loads(group.commands)
 
+                    ma = re.match(r"^/\S+\[CQ:at,qq=(\d+)\]", receive["message"])

```

Comment:
I don't think this regular expression can match commands like `/bot info [CQ:at,qq=12322222]`.

Reply:
Yes, but it will match command like `/bot[CQ:at,qq=12322222] info`

Reply:
I don't think we should force users to use such a command pattern.

Reply:
I add this because Telegram can use this pattern, and I don't think users have to use this pattern, they can still use `/bot info` optionally, or if you means users might use `/bot info [CQ:at,qq=12322222]` too then i will change my pattern.